components/modelregistry/modelregistry.go

```diff
@@ -22,20 +22,20 @@ import (
 	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/cluster"
 	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/conversion"
 	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/deploy"
+	"github.com/opendatahub-io/opendatahub-operator/v2/pkg/metadata/labels"
 
 	_ "embed"
 )
 
-const DefaultModelRegistryCert = "default-modelregistry-cert"
+const (
+	DefaultModelRegistryCert = "default-modelregistry-cert"
+	ModelRegSelector         = "model-registry"
```

Comment:
can we use component name?

Reply:
thought to use Component name at first, but then it uses `model-registry-operator` also move type var to const.
if that is fine, i can do the change.

Reply:
I do not have a strong opinion, and yes, '-operator' confused me as well. But on the other hand, we have pretty defined components, and the label is designed for components, so I'm biased a bit to reuse it. @lburgazzoli ?

Reply:
we could have use `model-registry-operator` maybe it is easier, since all other ModelReg resource have this already used.
tbh, i feel  the value for ComponentName should be `model-registry` not` model-registry-operator` at first.

Reply:
> i feel the value for ComponentName should be `model-registry` not` model-registry-operator` at first.

Yep.